platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/StandardThreadFactoryBuilder.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.platform.util;
+
+import java.util.concurrent.ThreadFactory;
+import java.util.concurrent.atomic.AtomicLong;
+
+public class StandardThreadFactoryBuilder implements ThreadFactory {
+
+    public static ThreadFactory newThreadFactory(String threadNamePrefix) {
+        return newThreadFactory(threadNamePrefix, true, Thread.NORM_PRIORITY);
+    }
+
+    public static ThreadFactory newThreadFactory(String threadNamePrefix, boolean isDaemon, int priority) {
```

Comment:
`@Nullable` threadNamePrefix?

Reply:
Prefix is misleading since I only tack a number on the end. Can update.